tests/testsuite/messages.rs

```diff
@@ -48,6 +46,20 @@ pub fn raw_rustc_output(project: &Project, path: &str, extra: &[&str]) -> String
     result
 }
 
+fn redact_rustc_message(msg: &str) -> String {
+    let mut msg = msg
+        .replace("help:", "[HELP]")
+        .replace("note:", "[NOTE]")
+        .replace("warning:", "[WARNING]")
+        .replace("error:", "[ERROR]");
+
+    if cfg!(windows) {
+        msg = msg.replace("\\", "/");
+    }
```

Comment:
We do this for all platforms, not just windows

`snapbox` provides a `Filter` for this you can reuse

Reply:
I found that the slash replacement need  `[Data::is_path_set`](https://github.com/assert-rs/snapbox/blob/main/crates%2Fsnapbox%2Fsrc%2Fassert%2Fmod.rs#L124-L124) in condition to filter but no way for Data::filters to set `paths()` using test support crate. Or I misunderstood what you suggested?